projects/fal/src/fal/auth/__init__.py

```diff
@@ -191,13 +189,17 @@ def bearer_token(self) -> str:
         return "Bearer " + self.access_token
 
     @property
-    def teams(self) -> list[dict]:
-        if self._teams is None:
-            self._teams = _fetch_teams(self.bearer_token)
-        return self._teams
+    def accounts(self) -> list[dict]:
```

Comment:
i feel like `team` was clearer. Not clear why you are filtering out personal accounts in `accounts` now